packages/vaadin-text-field/src/vaadin-text-area.js

```diff
@@ -182,36 +209,11 @@ class TextAreaElement extends ElementMixin(TextFieldMixin(ControlStateMixin(Them
 
   /** @private */
   _updateHeight() {
-    const inputField = this.root.querySelector('[part=input-field]');
-    const scrollTop = inputField.scrollTop;
-    const input = this.inputElement;
-
-    const inputWidth = getComputedStyle(input).width;
-
-    const valueLength = this.value ? this.value.length : 0;
-    // Only clear the height when the content shortens to minimize scrollbar flickering.
-    if (this._oldValueLength >= valueLength) {
-      // Fix the input element width so its scroll height isn't affected by host's disappearing scrollbars
-      input.style.maxWidth = inputWidth;
-      input.style.height = 'auto';
-      // Avoid a jumpy Safari rendering issue
-      if (this.__safari) {
-        inputField.style.display = 'block';
-      }
-    }
-    this._oldValueLength = valueLength;
-
-    const inputHeight = input.scrollHeight;
-    if (inputHeight > input.clientHeight) {
-      input.style.height = inputHeight + 'px';
-    }
-
-    // Restore
-    input.style.removeProperty('max-width');
-    inputField.style.removeProperty('display');
-    inputField.scrollTop = scrollTop;
-
-    this._dispatchIronResizeEventIfNeeded('InputHeight', inputHeight);
+    /* https://css-tricks.com/the-cleanest-trick-for-autogrowing-textareas/ */
+    this._textAreaWrapper = this._textAreaWrapper || this.shadowRoot.querySelector('.textarea-wrapper');
+    this._textAreaWrapper.dataset.replicatedValue = this.inputElement.value;
```

Comment:
While handy, this solution doesn't work in IE11, right? If so, the fix can't be backported to V14 (LTS) where IE11 is still on the list of supported browsers.